editor/editor_themes.cpp

```diff
@@ -1453,6 +1453,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_stylebox("panel", "TabContainer", style_content_panel);
 
 	// Bottom panel.
+	theme->set_constant("h_separation", "BottomPanelButton", 6 * EDSCALE);
```

Comment:
```suggestion
	theme->set_type_variation("BottomPanelButton", "Button");
	theme->set_constant("h_separation", "BottomPanelButton", 6 * EDSCALE);
```

Using type variations for this is a good idea, but we should define it properly :)